### ISSUE TITLE
Will it be continued?

### ISSUE THREAD

**Issue by soTaddel** [20tab/UnrealEnginePython#678]
Hello :)

we are using your plugin excessively and are wondering if this project will be continued,
since epic now made their own Python support, as we all know. We are a little bit afraid...

Can you provide us any foresight?
We of course hope, that it will last!

Sebi

**Comment by rdeioris** [comment]
Hi, 20tab stopped investing in the project months ago (for obvious reasons). There are a bunch of companies paying for specific features that combined with my personal needs should keep the project more-or-less alive (but with a slower development cycle). Currently the epic plugins covers less than 10% of what this plugin does so it makes perfectly sense to continue it as a way for creating plugins, extensions and runtime-features in python. The 20tab's objective is to "sell" the project to another company willing to sponsor my effort. So if anyone want to do it, drop a mail to info at 20tab.com :)

**Comment by yobiminds** [comment]
I've sent an expression of interest, let see what happens.

**Comment by yobiminds** [comment]
After two weeks and some emails, got no answer (I tried also to contact Ikrima Elhassan)
Maybe we should start thinking about UEPython with a different approach.

**Comment by rdeioris** [comment]
@yobiminds I do not know if you are one of the companies/professionals that offered support in the past days. As i wrote via email, i am waiting for Epic to make a final decision on the future of the project. I should have a call with them in the next few days.

**Comment by yobiminds** [comment]
@rdeioris Yep (Roberto C.). Sorry, I didn't get any replay, but I am happy to hear that things are somehow in progress. Will wait in suspense. 👍

**Comment by RockShea** [comment]
Hello, is there an update on this?

**Comment by rdeioris** [comment]
@RockShea nope, i continue porting the plugin to the latest ue versions (4.23 is upcoming), but maybe it is better to drop a mail to epic directly to inform them about your interest in the project

**Comment by yobiminds** [comment]
I did mention your plugin in my proposal for Epic MegaGrants.

**Comment by rdeioris** [comment]
I submitted to Mega Grants too. The idea is to buy the plugin from 20tab (the old owner) and continue the development by focusing on integration with the epic effort (to avoid code duplication in the python plugin core)

**Comment by dfb** [comment]
Awesome! Thanks for the update; I'm following this with interest. We'd love to see UE4 support (a) modern Python and (b) usable at runtime, both of which are lacking in the engine right now, so this plugin has filled a big void for us.

It'd be terrific if they stepped in to support it in some capacity, so please let me know if there is any way we can help lobby / petition / nag Epic. :)

**Comment by darrickyee** [comment]
Any updates on this?  I'd definitely like to see this project continue, primarily for being able to script gameplay

**Comment by rdeioris** [comment]
@darrickyee still waiting for Mega Grant response. Pinging Epic should help too ;)

**Comment by darrickyee** [comment]
I'd be happy to nag them.  Can someone suggest the best way to do that?
